packages/server-wallet/src/handlers/updateChannel.ts

```diff
@@ -0,0 +1,84 @@
+import {Either, left, right, chain, map} from 'fp-ts/lib/Either';
+import {SignedStateWithHash, StateVariables, Outcome} from '@statechannels/wallet-core';
+import {pipe} from 'fp-ts/lib/function';
+import {ChannelId} from '@statechannels/client-api-schema';
+import {curry} from 'lodash';
+
+import {ProtocolAction, SignState} from '../protocols/actions';
+import {ChannelState} from '../protocols/state';
+
+type ChannelStateWithSupported = ChannelState & {
+  supported: SignedStateWithHash;
```

Comment:
We might as well make `latestSignedByMe` present

```suggestion
  supported: SignedStateWithHash;
  latestSignedByMe: SignedStateWithHash;
```